~

```diff
@@ -0,0 +1,7 @@
+Merge branch 'master' of https://www.github.com/Veriblock/vbk-ri-btc into xagau
+
+# Please enter a commit message to explain why this merge is necessary,
+# especially if it merges an updated upstream into a topic branch.
+#
+# Lines starting with '#' will be ignored, and an empty message aborts
+# the commit.
```

Comment:
This file looks like a commit template. Should it be in a repo?

Reply:
Fixed.